Header header

# Time by floating point
float64 time

#UTC Years                - [1900, 2089]
int32 year

#UTC Months	              - [   1,   12]
int32 mon

#Day of the month         - [   1,   31]
int32 day

#Hours since midnight     - [   0,   23]
int32 hour

#Minutes after the hour   - [   0,   59]
int32 min

#Seconds after the minute - [   0,   60]
int32 sec

#Hundredth part of second - [   0,   99]
int32 hsec